funnel/views/commentvote.py

```diff
@@ -187,15 +187,13 @@ def subscribe(self):
             db.session.commit()
             return {
                 'status': 'ok',
-                'message': _("You have successfully subscribed to this comment thread"),
+                'message': _("Successfully subscribed to this comment thread"),
```

Comment:
Just "Subscribed to this comment thread" should do. "Successfully" makes it sound like the user did something heroic.